ts/src/okx.ts

```diff
@@ -37,6 +37,8 @@ export default class okx extends Exchange {
                 'cancelOrder': true,
                 'cancelOrders': true,
                 'createDepositAddress': false,
+                'createMarketBuyOrderWithCost': true,
+                'createMarketSellOrderWithCost': false,
```

Comment:
@Dan-krm I think this might be possible 
![image](https://github.com/ccxt/ccxt/assets/43336371/5d17750f-b127-410d-b2ed-1cfce2d2513c)